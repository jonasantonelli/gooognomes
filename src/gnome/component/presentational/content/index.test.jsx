/**
 * @jest-environment jsdom
 */
import React from 'react';
import { shallow } from 'enzyme';
import Content from './index.jsx';

describe('Content Component', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Content />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain title class', () => {
    expect(wrapper.exists('.gnome__content')).toEqual(true);
  });

});
