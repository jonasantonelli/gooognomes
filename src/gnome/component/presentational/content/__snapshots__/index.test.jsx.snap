// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Component should match snapshot 1`] = `
<div
  className="gnome__content"
>
  <div
    className="gnome__filter"
  >
    <Filter
      handleClearFilter={[Function]}
      handleFilter={[Function]}
      maxAge={100}
      minAge={0}
      professions={Array []}
      range={
        Object {
          "max": 100,
          "min": 0,
        }
      }
    />
  </div>
  <div
    className="gnome__list"
  >
    <List
      data={Array []}
    />
  </div>
</div>
`;
